llvm/test/MC/Hexagon/hvx-nv-pair-reverse.s

```diff
@@ -0,0 +1,18 @@
+# RUN: llvm-mc -arch=hexagon -mv69 -mhvx -filetype=obj %s | \
```

Comment:
instead of `arch=hexagon` I think you want `triple=hexagon` as seems to be the trend in other tests.